tex/measure/measure-space.tex

```diff
@@ -351,17 +354,54 @@ \section{A hint of Banach-Tarski}
 in the next chapter.
 
 \section{Measurable functions}
+\label{sec:measurable_functions}
+\prototype{For $S \subseteq \Omega$, $\mathbf{1}_S \colon \Omega \to \RR$ is a measurable function if and
+only if $S$ is a measurable set.}
 In the past, when we had topological spaces,
 we considered continuous functions.
 The analog here is:
 \begin{definition}
 	Let $(X, \SA)$ and $(Y, \SB)$ be measurable spaces
 	(or measure spaces).
 	A function $f \colon X \to Y$ is \vocab{measurable}
-	if for any measurable set $S \subseteq Y$ (i.e.\ $S \in \SB$)
+	if for any open set $S \subseteq Y$ (i.e.\ $S \in \SB$)
```

Comment:
scary mistake. (although for the Borel measure, they're conveniently the same thing)

Reply:
I'm a bit confused by this. Is "measurable" not right?

https://en.wikipedia.org/wiki/Measurable_function

Reply:
For Borel measurable function it's the same, for Lebesgue measurable function it's not. I add two boxes below to explain it.